apps/src/bounce/bounce.js

```diff
@@ -752,6 +752,18 @@ Bounce.init = function (config) {
   config.enableShowCode = false;
   config.enableShowBlockCount = false;
 
+  if (
+    config.embed &&
+    config.level.markdownInstructions &&
+    !config.level.instructions
```

Comment:
Embedded level needs to have empty instructions to opt-in to markdown instructions?

Reply:
Yep; otherwise embedded levels try to render the regular instructions as the title, which conflicts with `centerEmbedded = false`. Also, the CSP instructions don't support toggling between short and long instructions.